app/screens/channel_info/channel_info.js

```diff
@@ -384,6 +403,16 @@ export default class ChannelInfo extends PureComponent {
                 togglable={true}
                 theme={theme}
             />
+            <View style={style.separator}/>
+            <ChannelInfoRow
+                action={this.handleIgnoreChannelMentions}
+                defaultMessage='Ignore @channel, @here, @all'
```

Comment:
Maybe @esethna can confirm but this sounds a bit off to me?  Should this have "and", something like "Ignore @channel, @here and @all"?
I can't find the design for that to check whether it's intended as short form due to small screen size.

Reply:
Matches the design given small screens, but thanks for checking